language_service/src/rename.rs

```diff
@@ -182,3 +187,87 @@ impl<'a> Handler<'a> for Rename<'a> {
         }
     }
 }
+
```

Comment:
Does this belong to this PR or did it sneak in from a merge?

Reply:
hmm... it showed up after doing a merge from main. I think I might need to rebase on main instead.

Reply:
Ah, looks like these got moved to another file at the same time that I updated them, and the conflict resolution ended up duplicating them. Should be fixed now.